helm/chaos-mesh/templates/collector-mysql.yaml

```diff
@@ -50,6 +50,14 @@ spec:
         - name: data
           persistentVolumeClaim:
             claimName: collector-mysql-pv-claim
+      {{- with .Values.dashboard.affinity }}
```

Comment:
Can you add `nodeSelector` field for this Deployment at here?

Reply:
Thanks for your comment : `nodeSelector` is added now.